pkg/build/util/generate.go

```diff
@@ -149,3 +157,14 @@ func SubstituteImageReferences(build *buildapi.Build, oldImage string, newImage
 		}
 	}
 }
+
+// SubstituteImageRepoReferences uses references to an image repository to set an actual image name
+func SubstituteImageRepoReferences(build *buildapi.Build, imageRepo kapi.ObjectReference, newImage string) {
+	switch {
```

Comment:
might as well leave it as we need to come back and add similar logic for docker and custom builds in the near future.